apps/organization/tests/test_models.py

```diff
@@ -0,0 +1,81 @@
+"""
+Organizations application tests
+"""
+from django.db import DataError, IntegrityError
+from django.test import TestCase
+
+from cms.api import create_page
+
+from ..factories import OrganizationFactory
+from ..models import Organization
+
+
+class OrganizationTestCase(TestCase):
+    """
+    Unit test suite to validate the behavior of Organization model
+    """
+    def test_organization_fields_name_required(self):
+        """
+        The `name` field should be required
+        """
+        with self.assertRaises(IntegrityError):
+            OrganizationFactory(name=None)
+
+    def test_organization_fields_name_max_length(self):
+        """
+        The `name` field should be limited to 200 characters
+        """
+        OrganizationFactory(name='a'*200)
+        with self.assertRaises(DataError):
+            OrganizationFactory(name='a'*201)
+
+    def test_organization_fields_code_required(self):
+        """
+        The `code` field should be required
+        """
+        with self.assertRaises(IntegrityError):
+            OrganizationFactory(code=None)
+
+    def test_organization_fields_code_unique(self):
+        """
+        The `code` field should be unique
+        """
+        OrganizationFactory(code='the-unique-code')
+        with self.assertRaises(IntegrityError):
+            OrganizationFactory(code='the-unique-code')
+
+    def test_organization_fields_code_max_length(self):
+        """
+        The `code` field should be limited to 100 characters
+        """
+        OrganizationFactory(code='a'*100)
+        with self.assertRaises(DataError):
+            OrganizationFactory(code='a'*101)
+
+    def test_organization_fields_not_required(self):
+        """
+        The `page` and `logo` are not required on the model
+        """
+        page = OrganizationFactory(page=None, logo=None)
+        self.assertTrue(page.pk is not None)
+        self.assertTrue(page.page is None)
+        self.assertFalse(bool(page.logo))
```

Comment:
`self.assertIsNone(page.logo)`?

Reply:
This one no. An imageField is not None and must be checked as a bool if I understood well.